advanced_html_css/accessibility/the_web_content_accessibility_guidelines_wcag.md

```diff
@@ -4,14 +4,16 @@ After the previous lesson, you should now have a better understanding of how imp
 
 Another source, one that goes over many different ways to help make websites more accessible, is the Web Content Accessibility Guidelines (WCAG). In this lesson we're going to skim the surface of the WCAG, just to familiarize you with them at a more basic level.
 
-### Learning outcomes
+### Lesson overview;
```

Comment:
For this error, the message may not be super great. The `Expected: ### Lesson overview;` is meant to be read more as `Expected: "### Lesson overview";`, so we want to omit the semi-colon here.